CONTRIBUTORS.md

```diff
@@ -841,6 +841,8 @@
 * Rohit Singh
 * Robin Varghese
 * Victoria Ottah
+* Bart Cieliński
+* alexkiro
```

Comment:
Have added you both @alexkiro @FullFatBart under these names, let me know if you'd like a different name. Thanks!

Also, if you're on Mastodon, let us know your handles – we'd like to give you a shout-out if you don't mind 😄

Reply:
`alexkiro` is perfect for me! I'm not on Mastodon, sorry.

Many thanks for all your work here!